cmd/wallet-js-sdk/src/gnap/client.js

```diff
@@ -6,7 +6,8 @@ SPDX-License-Identifier: Apache-2.0
 
 import axios from "axios";
 // TODO : Implement http message signature module in JS (https://github.com/trustbloc/agent-sdk/issues/348)
-// import { encode } from "js-base64";
+// import { encodeURI } from "js-base64";
```

Comment:
commented-out

Reply:
This was commented out before, it's not needed for now - it's a part of the TODO above. I just noticed we need to use `encodeURI` instead, so I updated it to not miss it later